packages/office-ui-fabric-react/src/components/Dialog/Dialog.Props.ts

```diff
@@ -9,126 +10,103 @@ export interface IDialog {
+  * Props to be passed through to Dialog Content
   */
-  onDismiss?: (ev?: React.MouseEvent<HTMLButtonElement>) => any;
+  contentProps?: IDialogContentProps;
 
   /**
-  * The subtext to display in the dialog.
+  * Props to be passed through to Modal
   */
-  subText?: string;
+  modalProps?: IModalProps;
 
   /**
-  * The title text to display at the top of the dialog.
+  * A callback function for when the Dialog is dismissed from the close button or light dismiss. Can also be specified separately in content and modal.
   */
-  title?: string;
-}
-
-export interface IDialogProps extends React.Props<Dialog>, IWithResponsiveModeState, IAccessiblePopupProps {
-  /**
-   * Optional callback to access the IDialog interface. Use this instead of ref for accessing
-   * the public methods and properties of the component.
-   */
-  componentRef?: (component: IDialog) => void;
+  onDismiss?: (ev?: React.MouseEvent<HTMLButtonElement>) => any;
 
   /**
   * Whether the dialog is displayed.
   * @default false
+  * @deprecated Pass through via dialogModalProps instead
   */
   isOpen?: boolean;
 
   /**
-  * The type of Dialog to display.
-  * @default DialogType.normal
-  */
-  type?: DialogType;
-
-  /**
   * Whether the overlay is dark themed.
   * @default true
+  * @deprecated Pass through via dialogModalProps instead
   */
   isDarkOverlay?: boolean;
 
   /**
-  * A callback function for when the Dialog is dismissed from the close button or light dismiss, before the animation completes.
-  */
-  onDismiss?: (ev?: React.MouseEvent<HTMLButtonElement>) => any;
-
-  /**
    * A callback function which is called after the Dialog is dismissed and the animation is complete.
+   * @deprecated Pass through via dialogModalProps instead
    */
   onDismissed?: () => any;
 
   /**
-  * The title text to display at the top of the dialog.
-  */
-  title?: string;
-
-  /**
-  * The subtext to display in the dialog.
-  */
-  subText?: string;
-
-  /**
   * Whether the dialog can be light dismissed by clicking outside the dialog (on the overlay).
   * @default false
+  * @deprecated Pass through via dialogModalProps instead
   */
   isBlocking?: boolean;
 
   /**
    * Optional class name to be added to the root class
+   * @deprecated Pass through via dialogModalProps instead
    */
   className?: string;
 
   /**
   * Optional override for container class
+  * @deprecated Pass through via dialogModalProps instead
   */
   containerClassName?: string;
 
   /**
-  * Optional override content class
-  */
-  contentClassName?: string;
-
-  /**
    * A callback function for when the Dialog content is mounted on the overlay layer
+   * @deprecated Pass through via dialogModalProps instead
    */
   onLayerDidMount?: () => void;
 
   /**
-   * Deprecated at 0.81.2, use 'onLayerDidMount' instead.
-   * @deprecated
-   */
-  onLayerMounted?: () => void;
```

Comment:
Ok, so funny story.  onLayerMounted was pulled in via `let { onLayerMounted } = this.props;` but it was never actually used in the code :)

Regardless, I put it back so that the signature didn't change, even if the prop was already removed from the component.